Migration.md

```diff
@@ -6,6 +6,13 @@
 * **Replacement**: `gz` namespace
 * **Deprecation**: `ignition/...` headers
 * **Replacement**: `gz/...` headers
+* **Deprecation**: `ignitionVersion`
+* **Replacement**: `gzVersion()`
```

Comment:
This can be removed, no need to tick-tock the private function

Reply:
https://github.com/gazebosim/gz-fuel-tools/pull/259/commits/59add5051848659b9c291109fafe3b429868b612